apis/v1alpha1/instrumentation_webhook_test.go

```diff
@@ -92,6 +92,53 @@ func TestInstrumentationValidatingWebhook(t *testing.T) {
 				},
 			},
 		},
+		{
+			name: "pollingIntervalMs is not a number",
+			err:  "invalid pollingIntervalMs: abc",
+			inst: Instrumentation{
+				Spec: InstrumentationSpec{
+					Sampler: Sampler{
+						Type:     JaegerRemote,
```

Comment:
Can you add duplicate tests for ParentJaegerRemote?

Reply:
or just define a separate matrix for  Jaeger samplers

Reply:
I can do that. I thought they would be redundant since there is no difference.